using_packages/workflows.rst

```diff
@@ -21,39 +21,44 @@ When working with a  single configuration, your conanfile will be quite simple a
 
 .. code-block:: bash
 
-    $ git clone https://github.com/memsharded/example-hello.git
-    $ conan install ./example-hello --build=missing --install-folder example-hello-build
+    $ git clone https://github.com/memsharded/example-poco-timer
+    $ conan install ./example-poco-timer --install-folder=example-poco-build
 
 This will result in the following layout:
 
 .. code-block:: text
 
-    example-hello-build
-      conaninfo.txt
-      conanbuildinfo.txt
-      conanbuildinfo.cmake
-    example-hello
-      conanfile.txt
-      CMakeLists.txt  # If using cmake, but can be Makefile, sln...
-      main.cpp
+    example-poco-build
+        conaninfo.txt
+        conanbuildinfo.txt
+        conanbuildinfo.cmake
+    example-poco-timer
+        CMakeLists.txt  # If using cmake, but can be Makefile, sln...
+        LICENSE
+        README.md
+        conanfile.txt
+        timer.cpp
 
 Now you are ready to build:
 
 .. code-block:: bash
 
-    $ cmake ../example-hello -G "Visual Studio 14 Win64"  # or other generator
+    $ cd example-poco-build
+    $ cmake ../example-poco-timer -G "Visual Studio 15 Win64"  # or other generator
     $ cmake --build . --config Release
-    $ ./bin/greet
+    $ ./bin/timer
 
 We have created a separate build configuration of the project without affecting the original
-source directory in any way. The benefit is that we can freely experiment with the configuration, and, if necessary, erase the build folder, and rerun the build with a new configuration with different settings:
+source directory in any way. The benefit is that we can freely experiment with the configuration, and, if necessary, erase the build folder, and rerun the build with a new configuration with different settings,
```

Comment:
"... we can freely experiment with the configuration: We can clear the build folder and build another. For example, changing the build type to Debug:"